modules/op-constructing-pipelines-using-pipeline-builder.adoc

```diff
@@ -5,41 +5,53 @@
 [id="op-constructing-pipelines-using-pipeline-builder_{context}"]
 = Constructing Pipelines using the Pipeline Builder
 
-In the *Developer* perspective of the console, you can use the *Add* -> *Pipeline* -> *Pipeline Builder* option to:
+In the *Developer* perspective of the console, you can use the *+Add* -> *Pipeline* -> *Pipeline Builder* option to:
```

Comment:
The _+_ looks like a graphic in the UI. While we don't have any official guidance on this I can find, IBM states to avoid punctuation.

Reply:
I agree with your point although we had a task for the same to change "Add" to "+Add" as the UI element is also "+Add" 
So we need to keep it like this only.

Reply:
Okay.